.golangci.yml

```diff
@@ -59,8 +68,8 @@ issues:
         - goimports
         - govet
         - stylecheck
-  max-same-issues: 50
-
+  max-same-issues: 10000
+  max-issues-per-linter: 10000
```

Comment:
Just so that when linting we can see all of the issues instead of a subset.

Reply:
This also makes CI job slower because it doesn't stop after 50 issues. I think it's reasonable to expect a developer to run `make lint` herself to ensure that all the errors were fixed.

Reply:
if you use VScode, and you are tracking the issues in the problems tab, it's a bit infuriating.  The number never goes down unless it's configured as above. 

You **think** you're down to _ issues, press save and you're back at 500.